### PR TITLE
the date of the post should be the date you create it

### PR DESCRIPTION
**Pull request by cloud-fan**
After talking with @rxin offline, we think it's confusing to use the RC vote pass date as the date of the post. People may be wondering why they didn't see the post before, as the date is usually many days before.

This PR tries to make the policy clear: The date of the post should be the date you create it. This is the most intuitive IMO.

This PR also improves the release notes a bit, to mention that some of the known issues are already fixed in the next release.

### CODE REVIEW

**Comment by cloud-fan** [comment]
cc @dongjoon-hyun @rxin @srowen

File: release-process.md [hunk 1]
```diff
@@ -259,7 +259,8 @@ Then, create the release notes. Go to the
 <a href="https://issues.apache.org/jira/projects/SPARK?selectedItem=com.atlassian.jira.jira-projects-plugin:release-page">release page in JIRA</a>,
 pick the release version from the list, then click on "Release Notes". Copy this URL and then make a short URL on
 <a href="https://s.apache.org/">s.apache.org</a>, sign in to your Apache account, and pick the ID as something like
-`spark-2.1.2`. Create a new release post under `releases/_posts` to include this short URL.
+`spark-2.1.2`. Create a new release post under `releases/_posts` to include this short URL. The date of the post should
+be the date you create it.
```
Comment (cloud-fan) [inline release-process.md:263]: I put it here so future releases can just follow it.

**Review by rxin — APPROVED** [review]

**Comment by srowen** [comment]
I agree with that take. The VOTE passing is just an internal procedural step. The release date is when it's fully available to end users, which isn't the same thing.

File: releases/_posts/2020-06-18-spark-release-3-0-0.md [hunk 2]
```diff
@@ -315,18 +315,18 @@ Please read the [migration guide](https://spark.apache.org/docs/3.0.0/sparkr-mig
...
 
     Note that if you use S3AFileSystem, e.g. (“s3a://bucket/path”) to access S3 in S3Select or SQS connectors, then everything will work as expected. ([SPARK-30968](https://issues.apache.org/jira/browse/SPARK-30968))
 
-  - Parsing day of year using pattern letter 'D' returns the wrong result if the year field is missing. This can happen in SQL functions like `to_timestamp` which parses datetime string to datetime values using a pattern string. ([SPARK-31939](https://issues.apache.org/jira/browse/SPARK-31939))
-  - Join/Window/Aggregate inside subqueries may lead to wrong results if the keys have values -0.0 and 0.0. ([SPARK-31958](https://issues.apache.org/jira/browse/SPARK-31958))
-  - A window query may fail with ambiguous self-join error unexpectedly. ([SPARK-31956](https://issues.apache.org/jira/browse/SPARK-31956))
+  - Parsing day of year using pattern letter 'D' returns the wrong result if the year field is missing. This can happen in SQL functions like `to_timestamp` which parses datetime string to datetime values using a pattern string. This is fixed in the next release 3.0.1. ([SPARK-31939](https://issues.apache.org/jira/browse/SPARK-31939))
```
Comment (gatorsmile) [inline releases/_posts/2020-06-18-spark-release-3-0-0.md]: => This will be fixed in the next release 3.0.1.

**Comment by dongjoon-hyun** [comment]
Hi, @cloud-fan , @rxin , @gatorsmile .
This PR should change news date only~
- 2.4.4 (https://github.com/apache/spark-website/pull/217/files#diff-2e7c596ac11621f7b55e3299dd80f1b3R25)
- 2.4.5 (https://github.com/apache/spark-website/pull/261#discussion_r376745966)
- 2.3.4 (https://github.com/apache/spark-website/pull/221/files#diff-2e7c596ac11621f7b55e3299dd80f1b3R26)

**Comment by cloud-fan** [comment]
It's not a Spark PR, and I don't think the Spark PR policies apply here... We just do what's the most convenient.

**Comment by dongjoon-hyun** [comment]
Please see the 2.3.4/2.4.5 and old practices.

**Review by dongjoon-hyun — CHANGES_REQUESTED** [review]
We had better be consistent.

**Comment by srowen** [comment]
Meh, I'm sure we have roughly the same review practices here. I'd also generally separate concerns. In practice, it's also OK to have a 'misc minor edits' PR. I know we want to make sure it's fixed up ASAP now that 3.0 is rolling out the door, announced. So probably no big deal to just make this one omnibus last minute tweak change.

**Comment by dongjoon-hyun** [comment]
If you think so, okay. @srowen .

**Review by dongjoon-hyun — APPROVED** [review]

File: releases/_posts/2020-06-18-spark-release-3-0-0.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ meta:
   _wpas_done_all: '1'
 ---
 
-Apache Spark 3.0.0 is the first release of the 3.x line. Apache Spark 3.0 builds on many of the innovations from Spark 2.x, bringing new ideas as well as continuing long-term projects that have been in development. With the help of tremendous contributions from the open-source community, this release resolved more than 3400 tickets as the result of contributions from over 440 contributors.
+Apache Spark 3.0.0 is the first release of the 3.x line. The vote passed on the 10th of June, 2020. This release is based on git tag `v3.0.0` which includes all commits up to June 10. Apache Spark 3.0 builds on many of the innovations from Spark 2.x, bringing new ideas as well as continuing long-term projects that have been in development. With the help of tremendous contributions from the open-source community, this release resolved more than 3400 tickets as the result of contributions from over 440 contributors.
```
Comment (cloud-fan) [inline releases/_posts/2020-06-18-spark-release-3-0-0.md:14]: This change is necessary as we need to mention the vote pass date somewhere.

**Pull request merged**